tests/pages/catalog/packages/package/PackageTab-cy.js

```diff
@@ -110,6 +110,55 @@ describe("Package Detail Tab", function() {
         expect(result).to.equal(true);
       });
     });
+
+    context("Last updated", function() {
+      it("Shows a label, but no warnings and errors for recently updated packages", function() {
+        cy.get(".last-updated-package").should("have.length", 2);
+        cy.get(".last-updated-package-danger").should("not.exist");
+        cy.get(".last-updated-package-warning").should("not.exist");
+        cy.get(".page-body-content .package-action-buttons button")
+          .contains("Review & Run")
+          .click();
+        cy.get(".last-updated-package");
+        cy.get(".last-updated-package-danger").should("not.exist");
+        cy.get(".last-updated-package-warning").should("not.exist");
+      });
+
+      it("Shows a red warning if the package has not been updated for over a year", function() {
+        cy.configureCluster({
+          mesos: "1-task-healthy",
+          universePackages: "older"
+        }).visitUrl({ url: "/catalog/packages/marathon?version=1.4.6" });
+
+        cy.get(".last-updated-package-danger").contains(
+          "Last updated: 2018-01-24"
+        );
+
+        cy.get(".last-updated-package-danger").should("have.length", 2);
+        cy.get(".page-body-content .package-action-buttons button")
+          .contains("Review & Run")
+          .click();
+        cy.get(".last-updated-package-danger").contains("2018-01-24");
+      });
+
+      it("Shows a yellow warning if the package has not been updated for over a six months", function() {
+        cy.configureCluster({
+          mesos: "1-task-healthy",
+          universePackages: "old"
+        }).visitUrl({ url: "/catalog/packages/marathon?version=1.4.6" });
+
+        cy.get(".last-updated-package-warning").contains(
+          "Last updated: 2019-01-24"
```

Comment:
I think this will eventually fail, since we use the current date when checking. However, I cannot think of a solution now, we will have to change the config after the 24th of January 2020.